wiki/CreatingGovernments.md

```diff
@@ -7,7 +7,7 @@ The [syntax](DataFormat#grammar-specifications) for the definition of a governme
 ```html
 government <name>
 	"display name" <other-name>
-	swizzle <value#>
+	swizzle <value# | name>
```

Comment:
Isn't this always a name now?

Reply:
It's technically backwards-compatible with all previous swizzle values... but I'll change it.